libs/math/src/math/linalg/blas/gemm_nn_vector.cpp

```diff
@@ -124,6 +124,16 @@ template class Blas<float, Signature(_C <= _alpha, _A, _B, _beta, _C),
                     Computes(_C <= _alpha * _A * _B + _beta * _C),
                     platform::Parallelisation::VECTORISE>;
 
+// template class Blas<fetch::fixed_point::FixedPoint<16, 16>, Signature(_C <= _alpha, _A, _B,
```

Comment:
left in commented out code?

Reply:
no, it's just that gcc8 complains on those particular definitions, other compilers pass.

Reply:
fixed.